assets/queries/ansible/aws/rds_auto_minor_version_upgrade_disabled/query.rego

```diff
@@ -0,0 +1,113 @@
+        "keyActualValue": "AWS RDS instance feature auto_minor_version_upgrade is false"
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+
+    object.get(task["community.aws.rds_instance"], "auto_minor_version_upgrade", "undefined") == "undefined"
+
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.{{community.aws.rds_instance}}", [task.name]),
+        "issueType": "MissingAttribute",
+        "keyExpectedValue": "AWS RDS instance feature auto_minor_version_upgrade should be true",
+        "keyActualValue": "AWS RDS instance feature auto_minor_version_upgrade is false"
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+
+    isAnsibleFalse(task.rds.auto_minor_version_upgrade)
+
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.rds.auto_minor_version_upgrade", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": "AWS RDS instance feature auto_minor_version_upgrade should be true",
+        "keyActualValue": "AWS RDS instance feature auto_minor_version_upgrade is false"
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+
+    object.get(task.rds, "auto_minor_version_upgrade", "undefined") == "undefined"
+
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.rds", [task.name]),
+        "issueType": "MissingAttribute",
+        "keyExpectedValue": "AWS RDS instance feature auto_minor_version_upgrade should be defined",
+        "keyActualValue": "AWS RDS instance feature auto_minor_version_upgrade is undefined"
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+
+    isAnsibleFalse(task["community.aws.rds"].auto_minor_version_upgrade)
+
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.{{community.aws.rds}}.auto_minor_version_upgrade", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": "AWS RDS instance feature auto_minor_version_upgrade should be true",
+        "keyActualValue": "AWS RDS instance feature auto_minor_version_upgrade is false"
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+
+    object.get(task["community.aws.rds"], "auto_minor_version_upgrade", "undefined") == "undefined"
+
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.{{community.aws.rds}}", [task.name]),
+        "issueType": "MissingAttribute",
+        "keyExpectedValue": "AWS RDS instance feature auto_minor_version_upgrade should be true",
+        "keyActualValue": "AWS RDS instance feature auto_minor_version_upgrade is false"
+    }
+}
```

Comment:
I don't think the resources being targeted in this section exist, so this could be removed

Reply:
Removed!